articles/working-with-lambda-expressions-and-anonymous-functions/index.md

```diff
@@ -0,0 +1,114 @@
+### Introduction
+
+Lambda expressions can be challenging to understand. Types that look like this: `(Int) -> String` are not intuitive to many developers. This article aims to simplify the concept behind lambda expressions and how to use thm.
+
+### Prerequisites
+
+1. A basic understanding of the Kotlin programming language.
+2. Either IntelliJ IDEA or access to [Kotlin Playground](https://play.kotlinlang.org/)
```

Comment:
I noticed you were adding links directly, in the other articles make sure you use the markdown styling for links.